keyboards/wavtype/foundation/keymaps/via/via.json

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
This file should be deleted as via config files live in another repo.

Reply:
thanks, missed that.

Reply:
do i need to make a new pull request after the changes are made?